### DISCUSSION TITLE
Sugestion - Validation alert

### DISCUSSION THREAD

**Discussion by rfos** [directus/directus#12890] [Feature Requests]
The validation message shows the string used the render the translated message when a user fills up the wrong data in a validated.

![2022-04-20 20_09_15-Editing Item in Mobile Device and 9 more pages - Work - Microsoft​ Edge](https://user-images.githubusercontent.com/217889/164299989-6772ec06-a9b0-410c-915f-a5be0b00e149.png)

**Comment by azrikahar** [comment]
Thanks for bringing this up! Submitted a fix in #12916.
